docs/appendices/spin.md

```diff
@@ -17,3 +17,5 @@ The Spin-2 system developed by QuTech supports the following subset of the cQASM
 * All other libqasm 1.x language structures are not supported.
 * No `prep_X` and `measure_X` instructions are allowed, the lls will init all supported qubits before circuit execution
```

Comment:
So measure_X can be removed here (?)

Reply:
yes, currently the `measure_X` instruction is not part of the language specification.